### ISSUE TITLE
Unity3d link to login to Unity Hub did not work

### ISSUE THREAD

**Issue by lglista** [brave/brave-browser#20056]
## Description 
Setting up Unity Hub requires you to go through an in browser process to login. The browser is then supposed to send the login confirmation to the Unity Hub application. In Brave this process hangs upon logging in on the browser. In Firefox, this process goes through without a problem and lets me login to Unity Hub.

## Steps to Reproduce
   1.  Download Unity Hub
   2.  Create an account / Login using the Unity Hub application
   3.  Notice the browser hang

## Actual result:
Browser hangs on a blank screen that matches the background of the previous webpage

## Expected result:
Data is sent to Unity Hub to allow login confirmation

## Reproduces how often: 
I tried copy and pasting the link and also going through the regular steps to get this to work about 10 times and it failed every time.

## Brave version (brave://version info)
Brave | 1.32.115 Chromium: 96.0.4664.93 (Official Build) (64-bit)
Revision | 17531e0a70b4f8108f2418e8b5117f465077710b-refs/branch-heads/4664@{#1229}
OS | Linux

## Other Additional Information:
- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? N/A
- Is the issue reproducible on the latest version of Chrome? N/A

## Miscellaneous Information:
This may have to do with there not being a default app to carry out this process set in Brave. When tried in Firefox it first asked me to set a default application for this process, then it automatically recognized Unity Hub as a possible choice and let me pick it.

**Comment by ryanbr** [comment]
Does allowing fingerprinting in shields help @lglista  ?

**Comment by lglista** [comment]
I tried this with brave shields completely off a few times, but I did not try it with Brave shields on and fingerprinting off

**Comment by ryanbr** [comment]
> Thats against what shields are for, isnt it?

Yes and No, some sites (like webgl) will need fingerprinting to get details on how to draw. But for the most part these are edge cases. The rest of the shield settings are still enabled for protection.
